code/socialDistribution/views.py

```diff
@@ -79,6 +93,8 @@ def register(request):
                     return render(request, 'user/register.html', context)
 
                 user = form.save()
+                user.is_active = False # admin must approve user from console 
```

Comment:
Can we add some kind of check here to see if we have the approval option on or off? From my understanding, this feature can be turned on/off.

Reply:
Sounds good. I look/ask on the forum in the meantime.